tests/clone-populator_test.go

```diff
@@ -366,7 +366,7 @@ var _ = Describe("Clone Populator tests", func() {
 		})
 
 		AfterEach(func() {
-			if snapshot != nil {
+			if snapshot == nil {
```

Comment:
Not sure what's happening here?

Reply:
It was a buggy check. We deleted the snapshot only if it was nil...